code/modules/client/client_procs.dm

```diff
@@ -104,12 +104,6 @@
 
 	..()	//redirect to hsrc.Topic()
 
-/client/proc/is_content_unlocked()
```

Comment:
Instead of removing this check, why not change how `unlock_content` is assigned (make it always return TRUE for now?), so we don't have to reinvent the wheel later?

Changing [`preferences.dm:121`](https://github.com/Birdtalon/OracleStation/blob/byond-sucks/code/modules/client/preferences.dm#L121) to `unlock_content = TRUE` would have the same effect.